README.md

```diff
@@ -37,6 +37,7 @@ Cargo requires the following tools and packages to build:
 * `cmake`
 * `pkg-config`
 * OpenSSL headers (`libssl-dev` package on ubuntu)
+* libgit2
```

Comment:
Here's what I was seeing originally:
https://gist.github.com/seth/ef06472d873885075c4a

However, I believe I tracked this down to having accidentally had an alternate pkg-config binary on my PATH.

I just tried `brew uninstall libgit2 && make clean && make` and it built fine. So I have a feeling the funkiness of my environment was the issue. Will remove this note.